sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/RelationshipSamples.cs

```diff
@@ -3,126 +3,193 @@
-            Response<string> createSourceDtResponse = await client.CreateDigitalTwinAsync(sourceDtId, sourceDtPayload);
-            Console.WriteLine($"Created a digital twin with Id {sourceDtId}. Response status: {createSourceDtResponse.GetRawResponse().Status}.");
+            string digitalTwin1Payload = JsonSerializer.Serialize(digitalTwin1);
+            Response<string> createDigitalTwin1Response = await client.CreateDigitalTwinAsync("sampleTwin1Id", digitalTwin1Payload);
+            Console.WriteLine($"Created a digital twin with Id sampleTwin1Id. Response status: {createDigitalTwin1Response.GetRawResponse().Status}.");
 
-            // Setup a digital twin instance to be used as the target of the relationship.
-            string targetDtId = await GetUniqueTwinIdAsync(SamplesConstants.TemporaryTwinPrefix, client);
-            var targetTwin = new BasicDigitalTwin
+            // Create a digital twin instance to be used as another end of a relationship.
+            var digitalTwin2 = new BasicDigitalTwin
             {
-                Id = targetDtId,
-                Metadata = { ModelId = modelId }
+                Id = "sampleTwin2Id",
+                Metadata = { ModelId = "dtmi:samples:TempModel;1" }
             };
 
-            string targetDtPayload = JsonSerializer.Serialize(targetTwin);
-            Response<string> createTargetDtResponse = await client.CreateDigitalTwinAsync(targetDtId, targetDtPayload);
-            Console.WriteLine($"Created a digital twin with Id {targetDtId}. Response status: {createTargetDtResponse.GetRawResponse().Status}.");
-
-            // Setup a relationship between the source and target digital twins
-            string relationshipId = await GetUniqueRelationshipIdAsync(sourceDtId, SamplesConstants.TemporaryRelationshipIdPrefix, client);
+            string digitalTwin2Payload = JsonSerializer.Serialize(digitalTwin2);
+            Response<string> createDigitalTwin2Response = await client.CreateDigitalTwinAsync("sampleTwin2Id", digitalTwin2Payload);
+            Console.WriteLine($"Created a digital twin with Id sampleTwin2Id. Response status: {createDigitalTwin2Response.GetRawResponse().Status}.");
 
-            #region Snippet:DigitalTwinsSampleCreateRelationship
+            // Create a relationship between sampleTwin1Id and sampleTwin2Id using the BasicRelationship serialization helper.
+            #region Snippet:DigitalTwinsSampleCreateBasicRelationship
 
-            var basicRelationship = new BasicRelationship()
+            var basicRelationshipPayload = new BasicRelationship
             {
-                Id = relationshipId,
-                TargetId = targetDtId,
-                SourceId = sourceDtId,
-                Name = "related"
+                Id = "sampleRelationship1Id",
+                SourceId = "sampleTwin1Id",
+                TargetId = "sampleTwin2Id",
+                Name = "related",
```

Comment:
It would be best, for this sample, I think, if the relationship names weren't the same. We don't want to imply the relationships have to be the same.

Maybe we pick some physical relational examples, like "contains" and "containedBy".

Reply:
Great idea.